python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -938,15 +938,16 @@ def _fetch_column_metadata(
 
     with adapter.connection_named(f"column_metadata_{dbt_resource_props['unique_id']}"):
         try:
-            relation = adapter.get_relation(
+            relation = adapter.Relation.create(
                 database=dbt_resource_props["database"],
                 schema=dbt_resource_props["schema"],
                 identifier=dbt_resource_props["name"],
             )
             cols: List[BaseColumn] = adapter.get_columns_in_relation(relation=relation)
         except Exception as e:
             logger.warning(
```

Comment:
if you add exc_info=True to the warning call here (and the other warnings that catch exceptions in this file) it will include the stack trace in the process logs and stdout tab

Reply:
ah right, good call

Reply:
Did this in https://github.com/dagster-io/dagster/pull/22641